src/build_edn/main.clj

```diff
@@ -1,15 +1,18 @@
 (ns build-edn.main
   (:require
    [aero.core :as aero]
-   [build-edn.core :as core]))
+   [build-edn.core :as core]
+   [clojure.java.io :as jio]))
```

Comment:
Could you use idiomatic namespace aliases?
https://github.com/bbatsov/clojure-style-guide#use-idiomatic-namespace-aliases

```suggestion
   [clojure.java.io :as io]))
```